BulkyWeb/Controllers/CategoryController.cs

```diff
@@ -16,5 +16,83 @@ public IActionResult Index()
             List<Category> objCategoryList = _db.Categories.ToList();
             return View(objCategoryList);
         }
+
+        public IActionResult Create()
+        {
+            return View();
+        }
+        [HttpPost]
+        public IActionResult Create(Category obj)
+        {
+            if (obj.Name == obj.DisplayOrder.ToString())
```

Comment:
Not sure what this is trying to achieve. Please clarify

Reply:
Do not merge until changes are fixed

Reply:
We use [HttpPost] if we are sending information to the server, that's why it's there.